crates/volta-core/src/inventory/serial.rs

```diff
@@ -200,6 +200,9 @@ pub struct PackageMetadata {
     pub versions: HashMap<String, PackageVersionInfo>,
     #[serde(rename = "dist-tags")]
     pub dist_tags: PackageDistTags,
+    // and now we can get engines in here, don't have to wait to parse the package.json is we don't
```

Comment:
Ooh, I missed this - I should be able to implement this, getting the data from the info returned by `npm view`

Reply:
Actually I'm going to remove this. We can get the engines data, but in the case of just a fetch (instead of install) we would need to persist this somewhere, which is currently not done. I'll leave things as-is for now, reading the engines from package.json once the install happens.